### PR TITLE
added Cart Management Page

### PR DESCRIPTION
**Pull request by omjadhav85**
image

### CODE REVIEW

**Review by Shivanshsharma01 — APPROVED** [review]
Clean code Good to merge

File: routes/cartPage.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://subtleui.netlify.app/styles/styles.css"
+    />
+    <script
```
Comment (dubeyaditya29) [inline routes/cartPage.html:11]: suggestion : script tag can  be kept at the bottom to reduce the loading time.

File: routes/cartPage.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+      <a href="#" class="nav-heading">Sneakoholics</a>
+      <input
+        type="text"
+        class="input"
+        placeholder="What do you want to wear today?"
+      />
+      <ul class="nav-actions">
+        <li class="nav-action-item">
+          <button class="btn btn-primary">
+            <a href="./logout.html">Logout</a>
```
Comment (dubeyaditya29) [inline routes/cartPage.html]: Keeping anchor tag inside button is not a good HTML practice. You can check this https://stackoverflow.com/questions/16280684/nesting-a-inside-button-doesnt-work-in-firefox

**Review by dubeyaditya29 — CHANGES_REQUESTED** [review]
Apart from the changes suggested the code looks clean and its good to merge.

File: routes/cartPage.html (hunk 2 above)
Comment (omjadhav85) [reply routes/cartPage.html]: Thanks Aditya! Did not know this before...made the changes.

**Comment by omjadhav85** [comment]
Thanks Shivansh and Aditya!

**Pull request merged**